src/edge/edgeManager.ts

```diff
@@ -372,6 +393,28 @@ export class EdgeManager {
         }
     }
 
+    private async updateRuntimeVersionInDeploymentTemplate() {
+        const pattern = `{${Constants.deploymentTsonPattern}}`;

```

Comment:
'J'?

Reply:
reused existing variable, will rename it, thanks for catching

---
In reply to: [602420877](https://github.com/microsoft/vscode-azure-iot-edge/pull/569#discussion_r602420877) [](ancestors = 602420877)